ui/src/components/yaml/index.tsx

```diff
@@ -377,35 +380,45 @@
                   />
```

Comment:
The div with class `yaml_content_diagnosisContent` has been renamed to `yaml_content_diagnosisBody`. This change seems to be part of restructuring the component's layout.